src/isMatch.js

```diff
@@ -0,0 +1,28 @@
+'use strict'
+
+/**
+ * Checks if `object` is a match with `source`.
+ *
+ * @param {Object} object The object to inspect.
+ * @param {Object} source The object of property values to match.
+ * @returns {boolean} Returns `true` if `object` is a match, else `false`.
+ */
+const isMatch = (object, source) => {
```

Comment:
Small edge case if source is undefined it breaks, the strange thing is that it returns true in lodash. 
```console
console.log(isMatch(1)) // breaks
console.log(_.isMatch(1)) // true
```